cloud.amazon/licenseinfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/cloud/amazon/resources/empty.war</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/amazon.png</file>
+  	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/running.png</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/terminated.png</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/tomcat.png</file>
+   	<file>src/org/netbeans/modules/cloud/amazon/ui/resources/waiting.png</file>
```

Comment:
Please fix indentation.

Reply:
Has corrected now the indentation. In view still it is not showing properly.
Plz check the original file
https://github.com/apache/incubator-netbeans/blob/732e41fe8185aa4286116fff4e4483d9eb207bf8/cloud.amazon/licenseinfo.xml